src/app/data_file_manager/data_file_manager.cpp

```diff
@@ -19,6 +21,52 @@ void DataFileManager::promptReceptionist(clinic_data::Workers::Receptionist* new
     new_receptionist->set_gender(receptionist_data->getGender());
 }
 
+void DataFileManager::promptVisit(clinic_data::VisitsData::Visit* new_visit, const std::shared_ptr<Visit>& visit_data)
+{
+    if (!visit_data->getDoctor().expired())
+    {
+        clinic_data::VisitsData::Doctor* doctor = new_visit->mutable_doctor();
+        doctor->set_name(visit_data->getDoctor().lock()->getName());
+        doctor->set_surname(visit_data->getDoctor().lock()->getSurname());
+        doctor->set_pesel(visit_data->getDoctor().lock()->getPesel());
+        doctor->set_gender(visit_data->getDoctor().lock()->getGender());
+    }
+    if (!visit_data->getPatient().expired())
+    {
+        clinic_data::VisitsData::Patient* patient = new_visit->mutable_patient();
+        patient->set_name(visit_data->getPatient().lock()->getName());
+        patient->set_surname(visit_data->getPatient().lock()->getSurname());
+        patient->set_pesel(visit_data->getPatient().lock()->getPesel());
+        patient->set_gender(visit_data->getPatient().lock()->getGender());
+    }
+    if (!visit_data->getRoom().expired())
+    {
+        clinic_data::VisitsData::Room* room = new_visit->mutable_room();
+        room->set_room_id(visit_data->getRoom().lock()->getRoomNumber());
+    }
+    new_visit->set_visit_information(visit_data->getVisitInformation());
+    auto treatments = visit_data->getTreatments();
+    for (const auto& treatment : treatments)
+    {
+        new_visit->add_treatments(toString(treatment));
+    }
+}
+
+bool DataFileManager::checkFieldsIfBlank(const std::shared_ptr<Visit>& visit)
```

Comment:
1. body of the function suggests better name like "isAnyVisitFieldUnavailable" or something like that.
2. instead of using if explicitly just use return:
   ```
return visit->getDoctor().expired() || visit->getPatient().expired() || visit->getRoom().expired() ||
        visit->getVisitInformation().empty() || visit->getTreatments().empty();
   ```